milli/src/search/query_tree.rs

```diff
@@ -146,6 +146,12 @@ impl fmt::Debug for Query {
 
 trait Context {
     fn word_docids(&self, word: &str) -> heed::Result<Option<RoaringBitmap>>;
+    fn word_pair_proximity_docids(
+        &self,
+        right_word: &str,
+        left_word: &str,
+        proximity: u8,
+    ) -> heed::Result<Option<RoaringBitmap>>;
```

Comment:
You should directly implement `word_pair_frequency` in the `TestContext` instead of creating this function.
```suggestion
```

Reply:
Removed as recommended